src/sentry/replays/usecases/ingest/issue_creation.py

```diff
@@ -32,6 +33,31 @@ def report_rage_click_issue_with_replay_event(
 
     selector = selector
     clicked_element = selector.split(" > ")[-1]
+    component_name = component_name
+    evidence = [
+        IssueEvidence(name="Clicked Element", value=clicked_element, important=False),
+    ]
+    evidence_data = {
+        # RRWeb node data of clicked element.
+        "node": node,
+        # CSS selector path to clicked element.
+        "selector": selector,
+    }
+
+    if component_name:
+        # component name is important
+        evidence.extend(
+            [
+                IssueEvidence(name="Selector Path", value=selector, important=False),
+                IssueEvidence(name="React Component Name", value=component_name, important=True),
+            ],
+        )
+        evidence_data.update({"component_name": component_name})
+    else:
+        # selector path is important
+        evidence.append(
+            IssueEvidence(name="Selector Path", value=selector, important=True),
+        )
```

Comment:
i think we'd still want to add selector path either way if there's a selector, and we'd also want to add selector path, but just have it not be important

Reply:
wait nevermind. sorry i'm having issues this afternoon lol

Reply:
`evidence` starts out with `clicked_element`! just appending extra stuff depending on whether there's a component